develop/dev-guide-sample-application-python-mysqlclient.md

```diff
@@ -1,292 +1,238 @@
+### Step 2: Install dependencies (including mysqlclient)
 
-```python
-import uuid
-from typing import List
-
-import MySQLdb
-from MySQLdb import Connection
-from MySQLdb.cursors import Cursor
-
-def get_connection(autocommit: bool = True) -> MySQLdb.Connection:
-    return MySQLdb.connect(
-        host="127.0.0.1",
-        port=4000,
-        user="root",
-        password="",
-        database="test",
-        autocommit=autocommit
-    )
+Run the following commands to install required packages for the sample app.
 
+```bash
+cd tidb-example-python;
+pip install -r requirement.txt
+```
 
-def create_player(cursor: Cursor, player: tuple) -> None:
-    cursor.execute("INSERT INTO player (id, coins, goods) VALUES (%s, %s, %s)", player)
+<Tip> If you encounter installation issues, please refer to the [mysqlclient official documentation](https://github.com/PyMySQL/mysqlclient#install). </Tip>
 
+### Step 3: Configure Connection Information
 
-def get_player(cursor: Cursor, player_id: str) -> tuple:
-    cursor.execute("SELECT id, coins, goods FROM player WHERE id = %s", (player_id,))
-    return cursor.fetchone()
+Connect to your TiDB cluster depending on the TiDB deployment option you've selected.
 
+<SimpleTab>
+<div label="TiDB Serverless">
 
-def get_players_with_limit(cursor: Cursor, limit: int) -> List[tuple]:
-    cursor.execute("SELECT id, coins, goods FROM player LIMIT %s", (limit,))
-    return cursor.fetchall()
+1. In the TiDB Cloud, select your TiDB Serverless cluster. Go to the **Overview** page, and click the **Connect** button in the upper right corner.
 
+2. Ensure the configurations in the confirmation window match your operating environment.
 
-def random_player(amount: int) -> List[tuple]:
-    players = []
-    for _ in range(amount):
-        players.append((uuid.uuid4(), 10000, 10000))
+    - Endpoint is set to **Public Endpoint**
+    - Connect With is set to **General**
+    - Operating System matches your environment.
 
-    return players
+    <Tip>If you are running in Windows Subsystem for Linux (WSL), switch to the corresponding Linux distribution.</Tip>
 
+3. Click **Create Password** to create a password.
 
-def bulk_create_player(cursor: Cursor, players: List[tuple]) -> None:
-    cursor.executemany("INSERT INTO player (id, coins, goods) VALUES (%s, %s, %s)", players)
+    <Tip>If you have created a password before, you can either use the original password or click **Reset Password** to generate a new one.</Tip>
 
+4. Run the following command to copy `.env.example` and rename it to `.env`:
 
-def get_count(cursor: Cursor) -> None:
-    cursor.execute("SELECT count(*) FROM player")
-    return cursor.fetchone()[0]
+    ```bash
+    cp .env.example .env
```

Comment:
After you install the Git, which is mentioned in the **prerequisites** part, it will come with a component called Git Bash. `cp` command works on this component. We can not cover all things, otherwise, the document will be so redundant.